test/src/pages/base.page.js

```diff
@@ -0,0 +1,48 @@
+const ActionHelper = require('wdio-common/helpers/components/action-helper.js');
+const FilesHelper = require('wdio-common/helpers/utils/file-helper.js');
+
+
+class BasePage {
```

Comment:
Sorry, I don't get what's the purpose of this BasePage for and why Welcome page should extend from it.
If they are separated pages, intended to perform different and unrelated actions, none of them should extend the other.
On the other hand, this breaks the pageObject structure as is for now since there is no "platform" param on the Base page.
Last but not least, "Base page" it's a meaningless name since it doesn't describe (with its name) the page goals

Reply:
reviewing again, the intent is to build the parent page to change the pageObject structure, correct?
if that's so, we may need to organise a bit more the code, for example, creating another folder to store parent pages. Anyhow, the parent pages names should be the same as the child ones

Reply:
According to what was discussed yesterday, an agreement was reached to have a base class or superclass (whatever we want to call it) in order to store all the methods that are generic and thus avoid duplication.

For this reason, this class has been created and those methods have been taken there. On the other hand, the "btnRightNavigationArrow" method is being consumed in a lower method called storeMasterKey() and that is why it is inherited to the subclass.